<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWindow</class>
 <widget class="QMainWindow" name="QuestionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="startQuestion">
          <property name="text">
           <string>开始答题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>时间设定:[min]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="timeBox">
          <property name="maximum">
           <number>16777215</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="value" stdset="0">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>题目个数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="questionCountBox"/>
        </item>
        <item>
         <widget class="QLabel" name="resultInfo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="clearBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/appFace/clearcode</normaloff>:/appFace/clearcode</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="exitBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/appFace/exit</normaloff>:/appFace/exit</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="preBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>上一题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="nextBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>下一题</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="questionlist"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
